rust/arrow/src/util/bit_util.rs

```diff
@@ -366,13 +371,13 @@ mod tests {
         assert_eq!(ceil(8, 8), 1);
         assert_eq!(ceil(9, 8), 2);
         assert_eq!(ceil(9, 9), 1);
-        assert_eq!(ceil(10000000000, 10), 1000000000);
-        assert_eq!(ceil(10, 10000000000), 1);
-        assert_eq!(ceil(10000000000, 1000000000), 10);
+        assert_eq!(ceil(10000000, 10), 1000000);
```

Comment:
Why the changes here?

Reply:
I will double check these changes. I made them in response to a test that was failing; however, I found it suspect at the time as well.

Reply:
The numbers are 64-bit, which won't be supported in wasm32

Reply:
I think wasm32 supports i64 and f64 https://webassembly.github.io/spec/core/syntax/types.html